src/modules/admin/components/forms/FormDefinitionTable.tsx

```diff
@@ -10,48 +11,53 @@ import {
   GetFormIdentifiersQuery,
   GetFormIdentifiersQueryVariables,
 } from '@/types/gqlTypes';
-
 export type Row = NonNullable<
   GetFormIdentifiersQuery['formIdentifiers']
 >['nodes'][0];
 
-const columns: ColumnDef<Row>[] = [
-  {
-    header: 'Form Title',
-    render: ({ displayVersion }) => displayVersion.title,
-    width: '300px',
-    linkTreatment: true,
-  },
-  {
-    header: 'Form Type',
-    render: ({ displayVersion }) => <FormTypeChip role={displayVersion.role} />,
-  },
-  {
-    header: 'Applicability Rules',
-    render: ({ displayVersion }) => displayVersion.formRules.nodesCount,
-  },
-  {
-    key: 'system',
-    render: ({ displayVersion }) =>
-      displayVersion.system && (
-        <Chip
-          label='System'
-          size='small'
-          variant='outlined'
-          sx={{ width: 'fit-content' }}
-        />
-      ),
-  },
-  // TODO ADD: # projects active in (based on all rules)
-  // TODO ADD: version
-  // TODO ADD: status
-  // TODO ADD: last updated
-];
-
 interface Props {
   queryVariables: GetFormIdentifiersQueryVariables;
 }
 const FormDefinitionTable: React.FC<Props> = ({ queryVariables }) => {
+  const columns: ColumnDef<Row>[] = useMemo(
+    () => [
+      {
+        header: 'Form Title',
+        render: ({ displayVersion }) => displayVersion.title,
+      },
+      {
+        header: 'Form Type',
+        render: ({ displayVersion }) => (
+          <FormTypeChip role={displayVersion.role} />
+        ),
+      },
+      {
+        key: 'status',
+        header: 'Form Status',
+        render: (identifier) => startCase(identifier.displayVersion.status),
+      },
+      {
+        header: 'Applicability Rules',
+        render: ({ displayVersion }) => displayVersion.formRules.nodesCount,
+      },
+
+      {
+        key: 'system',
+        header: 'Form Tags',
+        render: ({ managedInVersionControl }) =>
+          managedInVersionControl && (
+            <Chip
+              label='System Form'
```

Comment:
I think it could also be helpful to add a comment here about how we decided not to use "system" to refer to this concept internally in the code, bc we have other places where "system" means something different, but we _did_ decide it was the most client-friendly representation of this concept and that's the reason for the language discrepancy

Reply:
added comment